app/javascript/pages/Settings/Password.tsx

```diff
@@ -0,0 +1,11 @@
+import { usePage } from "@inertiajs/react";
+import React from "react";
+import { cast } from "ts-safe-cast";
+
+import { default as PasswordPageComponent, type PasswordPageProps } from "$app/components/Settings/PasswordPage";
+
+export default function PasswordPage() {
+  const props = cast<PasswordPageProps>(usePage().props);
+
+  return <PasswordPageComponent {...props} />;
```

Comment:
We should inline the content of PasswordPageComponent here, same for the other inertia pages, you get the idea